src/language-css/parser-postcss.js

```diff
@@ -438,32 +439,32 @@ function parseNestedCSS(node) {
 }
 
 function parseWithParser(parser, text) {
+  const frontmatter = (function() {
```

Comment:
I personally would write this `frontMatter` and `front-matter`. What do you think?

Reply:
Makes no difference to me 🤷‍♂️